### PR TITLE
Fix toast colors

### PR DESCRIPTION
**Pull request by pivilartisant**
we are fixing toast text color for better readability

### CODE REVIEW

File: src/components/Toast/Toast.stories.tsx [hunk 1]
```diff
@@ -18,17 +18,19 @@ export const _Toast = {
...
-          toast.error('Euhh! This is an error message!');
-        }}
-      >
-        Create ERROR Toast
-      </Button>
-      <br />
-      <Button onClick={() => toast.success('Uuurraa! This is a success CSS')}>
-        Create SUCCESS Toast
-      </Button>
+      <div className="theme-dark">
```
Comment (thomas-senechal) [inline src/components/Toast/Toast.stories.tsx:21]: Why add this "theme-dark" here ? It just makes those 2 buttons white in the storybook🤔
Which is weird as the 3 others buttons are dark

Comment (Thykof) [reply src/components/Toast/Toast.stories.tsx:21]: in the storybook the toast is present 2 times in the DOM so the last theme win, there is a conflict. I could fix it last time.

**Comment by Thykof** [comment]
can you provide screenshots?

**Comment by pivilartisant** [comment]
Screenshot from 2024-05-21 15-09-51

**Review by Thykof — APPROVED** [review]

**Pull request merged**